src/test/shared/utilities/timeoutUtils.test.ts

```diff
@@ -54,11 +54,23 @@ describe('timeoutUtils', async () => {
         const checkTimerMs = 50
         it(`correctly reports an elapsed time with a ${marginOfError}ms margin of error`, async () => {
             const longTimer = new timeoutUtils.Timeout(300)
```

Comment:
What's actually under test here? Is it the `timeoutUtils.Timeout`

Reply:
The `elapsedTime` property of `timeoutUtils.Timeout`